migration/20191010-update-external-integration-mirrors.sql

```diff
@@ -11,26 +11,26 @@ DO $$
         EXCEPTION
             WHEN duplicate_table THEN RAISE NOTICE 'Warning: externalintegrationslinks already exists.';
         END;
-    END;
-$$; 
-
-ALTER TABLE ONLY externalintegrationslinks ADD CONSTRAINT externalintegrationslinks_pkey PRIMARY KEY (id);
+    
+        ALTER TABLE ONLY externalintegrationslinks ADD CONSTRAINT externalintegrationslinks_pkey PRIMARY KEY (id);
```

Comment:
Still not quite right -- this is inside the first BEGIN statement. I ran it on my dev database and got the warning as expected, but then an error, "multiple primary keys for table "externalintegrationslinks" are not allowed", which would have caused the script to fail.